packages/algob/src/types.ts

```diff
@@ -552,6 +574,10 @@ export interface Deployer {
    * Queries a stateful smart contract info from checkpoint. */
   getSSC: (nameApproval: string, nameClear: string) => SSCInfo | undefined
 
+  /**
+   * Queries a stateful smart contract info from checkpoint using key. */
+  getSSCFromKey: (key: string) => SSCInfo | undefined
```

Comment:
shouldn't this be a private method? User will use `getSSC` to retreive ssc info.

Reply:
I was saying from the user perspective, having both functions (`getSSC`, `getSSCFromKey`) looks confusing. yes, using `key.split("-")` could be wrong in some cases. Maybe we could move `registerCheckpoints` to deployer? If not, could we rename these functions ?

Reply:
I think renaming will be better, could you please suggest something ?
From user perspective I think it won't be confusing because getsscfromkey takes only one argument.

Reply:
how about `getSSCfromCPKey` or `getFromCPKey` ?

Reply:
`getSSCfromCPKey` works